augur/parse.py

```diff
@@ -47,16 +47,16 @@ def prettify(x, trim=0, camelCase=False, etal=None, removeComma=False):
     if 0 < trim < len(x):
         res = x[:trim] + "..."
 
-    if res in {'usvi', 'usa', 'uk'}:
-        res = res.upper()
-
     words = res.split('_')
 
     if camelCase:
         words = map(str.capitalize, words)
 
     res = ' '.join(words)
 
+    if res in {'usvi', 'usa', 'uk', 'Usvi', 'Usa', 'Uk'}:
```

Comment:
If you lowercase the string prior to searching the set of values (`if res.lower() in {'usvi', 'usa', 'uk'}:`), you won't need to maintain the extra entries for capitalized versions of the original strings.